repair/repair.cc

```diff
@@ -47,9 +47,33 @@
 
 #include <seastar/core/gate.hh>
 #include <seastar/util/defer.hh>
+#include <seastar/core/metrics_registration.hh>
 
 logging::logger rlogger("repair");
 
+struct node_ops_metrics {
+    seastar::metrics::metric_groups _metrics;
+    uint64_t bootstrap_nr_ranges_total{0};
+    uint64_t bootstrap_nr_ranges_finished{0};
+    uint64_t rebuild_nr_ranges_total{0};
+    uint64_t rebuild_nr_ranges_finished{0};
+    node_ops_metrics() {
+        namespace sm = seastar::metrics;
+        _metrics.add_group("node_ops", {
```

Comment:
I don't think that node_ops is not a good group name.

Reply:
How about node_operations? Those are for operations like bootstrap, rebuild, replace and so on.

Reply:
node_operations is better, though still not great, for me it sounds like the total number of operations.
@tzach do you have any input?
the `_nr_` is also not clear

Reply:
Or node_maintenance_operations? They are all under maintenance group.

Reply:
How about: 

scylla_node_maintenance_operations_bootstrap_finished_ranges
scylla_node_maintenance_operations_bootstrap_total_ranges

?

Reply:
I think it's better

Reply:
Good. I sent out v2 below.